### PR TITLE
Modify some message of bug 1528

### PR DESCRIPTION
**Pull request by hu-weihua**

### CODE REVIEW

File: xCAT-probe/subcmds/xcatmn [hunk 1]
```diff
@@ -510,9 +510,14 @@ sub check_disk {
     foreach $mountpoint (keys %mountpointinfo) {
         if ($mountpointinfo{$mountpoint}{need} > $mountpointinfo{$mountpoint}{available}) {
             foreach (@{ $mountpointinfo{$mountpoint}{mount} }) {
-                $msg .= "'$_' needs $dir_expectedspace_list{$_} G, ";
+                $msg .= "'$_' needs $dir_expectedspace_list{$_} GiB disk space, ";
+            }
+            my $mountmun = $#{$mountpointinfo{$mountpoint}{mount}} +1 ;
+            if($mountmun >1){
+                $msg .= "these directories are parts of file system '$mountpoint'. The free space available in directory '$mountpoint' is $mountpointinfo{$mountpoint}{available} GiB, it isn't enough.";
```
Comment (neo954) [inline xCAT-probe/subcmds/xcatmn]: Makes the first letter of a sentence a capital letter.

File: xCAT-probe/subcmds/xcatmn [hunk 2]
```diff
@@ -510,9 +510,14 @@ sub check_disk {
...
+            }else{
+                $msg .= "This directory is a part of file system '$mountpoint'. The free space available in directory '$mountpoint' is $mountpointinfo{$mountpoint}{available} GiB, it isn't enough.";
```
Comment (neo954) [inline xCAT-probe/subcmds/xcatmn]: Use `is not` instead of `isn't`.

File: xCAT-probe/subcmds/xcatmn (hunk 1 above)
Comment (hu-weihua) [inline xCAT-probe/subcmds/xcatmn]: It isn't the first letter of a sentence, below is the example output:

    '/var' needs 1 GiB disk space, '/install' needs 10 GiB disk space, '/tmp' needs 1 GiB disk space, these directories are parts of file system '/'. The free space available in directory '/' is 4 GiB, it isn't enough.

**Comment by neo954** [comment]
I will merge this one.

**Pull request merged**